src/flownet/templates/ahm_config.ert.jinja2

```diff
@@ -3,27 +3,26 @@
 INSTALL_JOB DELETE_IN_CURRENT_ITERATION ./DELETE_IN_CURRENT_ITERATION
 
 REFCASE <CONFIG_PATH>/SYNTHETIC_REFCASE
+{%- if ert_config.analysis: %}
+DEFINE <CONFIG_FILE>  ../brugge_parameters.yml
```

Comment:
This line should not be hard-coded.

Reply:
Good point, I had completely missed it!
Any ideas on how I could retrieve and pass along the name of the FlowNet config file?
I guess I will need to start from `_command_line.py` and pass it as an argument until `flownet.ert.create_ert_setup`.

Reply:
Is it an option to instead of passing along the configuration file, simply render the values/settings that `analysis_workflow` need into the ert config? This is somewhat similar to other values already (`MAX_RUNTIME`, `NUM_REALIZATIONS` etc.)

Reply:
Ok, all the values/settings are rendered in the ERT config now. And the changes have passed all CI tests just now